include/wx/generic/dirctrlg.h

```diff
@@ -116,6 +138,25 @@ class WXDLLIMPEXP_CORE wxGenericDirCtrl: public wxControl
     void OnItemActivated(wxTreeEvent &event);
     void OnSize(wxSizeEvent &event );
 
+    void OnRightClick(wxTreeEvent& event);
+
+    void AddRightClickMenuItem(wxString label, void(wxGenericDirCtrl::*function)(wxCommandEvent &));
```

Comment:
Prefer passing strings by reference, i.e. `const wxString&` and not by value (here and below).